SabreTools.Hashing/SpamSum/SpamSum.cs

```diff
@@ -352,5 +352,8 @@ private static int EliminateSequences(byte[] dst, int dstPtr, byte[] src, int sr
 
             return n;
         }
+
+        public static int FuzzyCompare(string? firstHash, string? secondHash)
```

Comment:
Nitpick: Please provide a summary for this, even if it's a copy of the one in `Comparisons`